Sources/TABResourceLoader/Extensions/Dictionary+Additions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Dictionary+Additions.swift
+//  TABResourceLoader
+//
+//  Created by John Sanderson on 30/01/2018.
+//  Copyright © 2018 Luciano Marisi. All rights reserved.
+//
+
+import Foundation
+
+public extension Dictionary where Key == String, Value == Any {
```

Comment:
presumably it's just left to the consumer to ensure their value type can be converted to a string? only works for primitive types.